first-30-minutes-of-the-writing-task-in-studieproeven.md

```diff
@@ -1,6 +1,6 @@
 # The first 30 minutes of the Writing task in Studieprøven
 
-Elena Sokolova, June 2nd 2020
+*Elena Sokolova, June 2<sup>nd</sup> 2020*
 
 In this blog post, you will learn about the rule of the first 30 minutes for the Writing task (Skriftlig fremstilling) in the written part of the Studieprøven exam. This rule will help you to complete the Writing task in the rest 2,5 hours. The three hours of the Writing task fly during the exam! Proper planning and understanding of the task questions and even data titles can save time for writing a coherent text on the topic. 
```

Comment:
Maybe it would be a good style to highlight all danish text in italics? Maybe this will make it easier to read when the languages are constantly switching?